full_report.go

```diff
@@ -28,7 +34,7 @@ func (r *fullReportScraper) UnmarshalYAML(unmarshal func(interface{}) error) err
 	}
 
 	r.setPuppetVersion(report.PuppetVersion)
-	r.setConfigTimestamp(report.ConfigurationVersion)
+	r.setConfigTimestamp(hash(report.ConfigurationVersion))
```

Comment:
why we want to convert `ConfigurationVersion` to hash?
Could you explain a use-case?